examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -73,37 +73,39 @@ class SSD(BaseEstimator, TransformerMixin):
         If not None (same as 'empirical'), allow regularization for
         covariance estimation. If float, shrinkage is used
         (0 <= shrinkage <= 1). For str options, estimator will be passed to
-        method to mne.compute_covariance().
+        method to :func:'mne.compute_covariance'.
```

Comment:
I think these have to be ` backticks  ` ....  see also below 😁

Reply:
great ! I changed them all !